src/busesApp/models.js

```diff
@@ -0,0 +1,36 @@
+import { BaseEntity, EntitySchema } from 'typeorm';
+
+export class Bus extends BaseEntity {
+	id;
+
+	plateNumber;
+
+	busType;
+
+	seats;
+}
+
+export const BusSchema = new EntitySchema({
+	name: 'Bus',
+	tableName: 'Bus',
+	target: Bus,
+	columns: {
+		id: {
+			primary: true,
+			generated: true,
+			type: 'int',
+		},
+		plateNumber: {
+			type: 'varchar',
+			unique: true,
+		},
+		busType: {
+			type: 'varchar',
+		},
+		seats: {
+			type: 'int',
+		},
+	},
+});
```

Comment:
shouldn't the bus also have a company it is registered in

Reply:
I forgot to include it. Will add it